### PR TITLE
Add Linux metadata files

### PR DESCRIPTION
**Pull request by orowith2os**

### CODE REVIEW

**Comment by stakira** [comment]
What exactly does Linux metadata refer to? Is it defined by who? Any document?

**Comment by orowith2os** [comment]
This metadata is for general Linux packages, such as Flatpak and RPM (think screenshots, app descriptions, etc.). It's defined by appstream, iirc.

File: Misc/Linux/com.github.stakira.OpenUtau.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        OpenUtau does not strike for Vocaloid compatibility, other than limited features.
+    </p>
+  </description>
+
+  <categories>
+    <category>Utility</category>
+    <category>Utau</category>
+  </categories>
+
+  <url type="homepage">https://github.com/stakira/OpenUtau</url>
```
Comment (stakira) [inline Misc/Linux/com.github.stakira.OpenUtau.appdata.xml:39]: www.openutau.com

File: Misc/Linux/com.github.stakira.OpenUtau.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>com.github.stakira.OpenUtau</id>
+
+  <name>OpenUtau</name>
+  <summary>OpenUtau is an open source editing environment for the UTAU community with a modern user experience.</summary>
```
Comment (stakira) [inline Misc/Linux/com.github.stakira.OpenUtau.appdata.xml:6]: Open singing synthesis platform / Open source UTAU successor

**Comment by stakira** [comment]
I see. Maybe put them under `Misc/AppStream` then.

**Comment by Vector-frame** [comment]
Does this mean that Open Utau also has a Linux version in Flatpak now?

**Comment by orowith2os** [comment]
@Vector-frame I've got a manifest and PR in the works for Flathub right now, I just need to find some free time to dedicate to setting it all up properly.

**Comment by Vector-frame** [comment]
Very nice of you! I'd love to help, both for this project and for UTSU, but unfortunately I don't have experience submitting to Flathub, let alone forking languages or programming itself.

**Comment by stakira** [comment]
closing due to in-activity

**Pull request closed without merging**